javascript/javascript1/week4/homework/voiceA.js

```diff
@@ -0,0 +1,112 @@
+let nameSave = "";
+let todos = [];
+const suffixFirst = "hello my name is ";
+const suffixSecond = "what is my name";
+const suffixThird = "add";
+const prefixThird = "to my todo";
+const suffixFourth = "remove";
+const prefixFourth = "from my todo";
+const suffixFifth = "what is on my to-do";
+const suffixSixth = "what day is it today";
+const suffixSeventh = "what is";
+let result;
+function getReply(command) {
+  const lowerCaseInput = command.toLowerCase();
+
+  //   const match = lowerCaseInput.match(suffixFirst
+  const startIndexFirst = lowerCaseInput.indexOf(suffixFirst); //   console.log(match);
+  const startIndexThird = lowerCaseInput.indexOf(suffixThird);
+  const startIndexFourth = lowerCaseInput.indexOf(suffixFourth);
+  const startIndexSeventh = lowerCaseInput.indexOf(suffixSeventh);
+  if (lowerCaseInput.includes(suffixFirst)) {
```

Comment:
I would consider re-attempting this using RegEx

An example of how the checks could be done are as so:

```
const patterns = {
  namePattern: /Hello, my name is (.+)/i,
  addPattern: /Add (.+) to my to do list\./i,
  removePattern: /Remove (.+) from my to do list\./i,
  showToDoPattern: /What is my to do\?/,
  mathPattern: /what is (\d+)\s*([+\-*\/])\s*(\d+)\?/i,
  tempPattern: /It is (\d+) degrees. What do I wear\?/,
  getNamePattern: /What is my name\?/i,
  datePattern: /What day is it today\?/i,
  timerPattern: /Set a timer for (\d+) min/i,
};
```

Then .test to see if it matches it.

You can [read more about regex here](https://www.w3schools.com/jsref/jsref_obj_regexp.asp)

It will require you Google each individual regex pattern but it comes in handy in the future!